Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn application:app
```

Comment:
You don't want this file. You are publish a docker image with the correct entrypoint so Heroku just has to run that.